main.cpp

```diff
@@ -20,67 +20,96 @@ You are going to write 10 UDTs in project3.
-        
-Sub Object 2: Instrument Cluster
-    Name 4 things you'll find on the:   Instrument Cluster
-        1) speed meter
-        2) gas meter
-        3) seatbelt notification
-        4) temperature
-    Name 3 things you can do with the:   Instrument Cluster
-        1) monitor speed
-        2) monitor gas level
-        3) monitor temperature
-    
-Sub Object 3: Environment Controls
-    Name 3 things you'll find on the:    Environment Controls
-        1) air conditioner on/off button
-        2) rear defrost button
-        3) temperature controls
-    Name 3 things you can do with the:   Environment Controls
-        1) turn on the air conditioner
-        2) adjust the temperature
-        3) activate the defroster
-
-Sub Object 4: Infotainment System
-    Name 3 things you'll find on the:    Infotainment System
-        1) radio button
-        2) phone button
-        3) clock
-    Name 3 things you can do with the:   Infotainment System
-        1) turn on the radio
-        2) activate the phone
-        3) tell the time
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1) seatbelt
-        2) leather material
-        3) seat adjuster
-    Name 2 things you can do with the:   Seat
-        1) fasten seatbelt
-        2) adjust the seat position
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - stores the amount of water used per week.
+        - stores amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) Gym
+5 properties:
+    1) number of treadmills
+    2) number of barbells
+    3) number of weight benches
+    4) number of protien bars
+    5) stores membership records
+3 things it can do:
+    1) provide excersize equiptment
+    2) provide space to workout
+    3) provide training programs
+ */
+
+/*
+Thing 2) Gas Station
+5 properties:
+    1) stores gas for pumps
+    2) number of gas pumps
+    3) stores product inventory
```

Comment:
`stores` implies an action.
you want properties. 
you can use `std::string` later when you define these properties as actual member variables, so feel free to use things like "product inventory list" instead
- [ ] revise